packages/std/src/testing/mock.rs

```diff
@@ -625,6 +637,12 @@ impl BankQuerier {
         result
     }
 
+    pub fn set_denom_metadata(&mut self, denom_metadata: &[DenomMetadata]) {
+        self.denom_metadata = denom_metadata.to_vec();
+        self.denom_metadata
+            .sort_unstable_by(|a, b| a.symbol.cmp(&b.symbol));
```

Comment:
Why not make this a BTreeMap in the first pace? This is also how the database in Cosmos SDK would store it. A KV store with denoms as key (more or less).

Reply:
Good point! Will do. btw, I'm not sure which field to use for this, so I just used symbol. Not sure what the sdk is actually using to construct the keys?

Reply:
It should be `base`. The `symbol` is for user interfaces. See https://github.com/cosmos/chain-registry/blob/master/testnets/junotestnet/assetlist.json#L20